src/Training.TruckWorld.Clone.Backend/TruckWorld.Infrastructure/Common/Notifications/Services/EmailSenderService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using FluentValidation;
+using TruckWorld.Application.Common.Notificaitons.Brokers;
+using TruckWorld.Application.Common.Notificaitons.Models;
+using TruckWorld.Application.Common.Notificaitons.Services;
+using TruckWorld.Domain.Enums;
+using TruckWorld.Domain.Extensions;
+
+namespace TruckWorld.Infrastructure.Common.Notifications.Services;
+
+public class EmailSenderService(IEnumerable<IEmailSenderBroker> emailSenderBrokers, IValidator<EmailMessage> validator)
+    : IEmailSenderService
+{
+    private IEnumerable<IEmailSenderBroker> _mailSenderBroker => emailSenderBrokers;
+    private IValidator<EmailMessage> _emailMessageValidator => validator;
```

Comment:
Don't need private fields, just use `emailSenderBrokers` that we got from primary constructor